studio/components/layouts/DatabaseLayout/DatabaseLayout.tsx

```diff
@@ -22,6 +22,9 @@ const DatabaseLayout: FC<Props> = ({ title, children }) => {
   const router = useRouter()
   const page = router.pathname.split('/')[4]
 
+  const vaultExtension = meta.extensions.byId('supabase_vault')
+  const isVaultEnabled = vaultExtension !== undefined && vaultExtension?.installed_version !== null
```

Comment:
I'm curious if the `?` can be removed in `vaultExtension?.installed_version` because of the `undefined` check beforehand. Can it be `null`?

Reply:
ahh right - vaultExtension can't be null, since its a derivation from a `find` - i think we can remove the undefined check actually. will do so!

Reply:
ahh right - yeah i think the ? can be removed in the second check here, will do so!